Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/Controls Toolkit/TransparentPanel.cs

```diff
@@ -0,0 +1,59 @@
+﻿#region MIT License
+/*
+ * MIT License
+ *
+ * Copyright (c) 2017 - 2024 Krypton Suite
```

Comment:
Just been added therefore start year is 2024

Reply:
Revised year